### PR TITLE
Remove Qt4 support: Part 1

### PR DESCRIPTION
**Pull request by mbr0wn**

### CODE REVIEW

**Comment by mbr0wn** [comment]
This is a bit of a hodgepodge of things. There's just so much that's broken w.r.t. Qt.

File: gr-qtgui/CMakeLists.txt [hunk 1]
```diff
@@ -29,15 +29,13 @@ include(GrPython)
 set(DESIRED_QT_VERSION 5 CACHE STRING "The Qt version to use (4 or 5)")
```
Comment (michaelld) [inline gr-qtgui/CMakeLists.txt]: Do we still need this line & the next line? Guessing they can now be removed & the variable replaced by "5" everywhere.

Comment (noc0lour) [reply gr-qtgui/CMakeLists.txt]: I agree, removing that will cleanup the cmake considerably.

File: gr-qtgui/docs/qtgui.dox [hunk 2]
```diff
@@ -129,7 +129,7 @@ The QT GUI blocks require the following dependencies.
 \li QtCore (version >= 4.4)
 \li QtGui (version >= 4.4)
 \li QtOpenGL (version >= 4.4)
-\li PyQt4 for Qt4 (version >= 4.4)
+\li PyQt5 for Qt5 (version >= 5.0)
```
Comment (noc0lour) [inline gr-qtgui/docs/qtgui.dox]: What's the lowest version we are able to test with?

File: gr-uhd/apps/uhd_fft [hunk 3]
```diff
@@ -54,7 +54,7 @@ except ImportError:
     from gnuradio.uhd.uhd_app import UHDApp
 
 
-class uhd_fft(gr.top_block, Qt.QWidget, UHDApp):
+class uhd_fft(UHDApp, gr.top_block, Qt.QWidget):
```
Comment (noc0lour) [inline gr-uhd/apps/uhd_fft]: Why the resort?

Comment (mbr0wn) [reply gr-uhd/apps/uhd_fft]: Something something Python3k. This defines an inheritance order (uhd_fft -> UHDApp -> top_block -> Widget), which means that UHDApp "inherits" from Widget. If it's the other way, then invoking Widget will call the ctor on UHDApp, which fails. The other way, it works.

**Comment by mbr0wn** [comment]
Sure removing `DESIRED_QT_VERSION` is much nicer. Will do that.

File: gr-qtgui/docs/qtgui.dox (hunk 2 above)
Comment (mbr0wn) [reply gr-qtgui/docs/qtgui.dox]: I don't know -- and the current code has no Qt version requirements whatsoever. Hence, I went down to the bottom.

**Comment by mbr0wn** [comment]
I resolved all of @noc0lour's complaints.

**Pull request merged**